.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### This PR adds | fixes:
+
+- 
+
```

Comment:
I'm a big fan of knowing how to Prepare for test/install, I guess it is kind of the "Implementation" description in the AM-lingo. Usually it's just a matter of running update-tool-stage.sh branch but sometimes we should have multiple PR's aligned on stage and sometimes even running some update-scripts on the data base. This description It could be part of the How to test, quite often we have several test cases and then it benefits from having an own section on preparation. What do you say?

Reply:
Added a section. @patrikgrenfeldt What do you think?